lib/vagrant-windows-domain/templates/runner.ps1.erb

```diff
@@ -99,7 +403,11 @@ if (Test-PartOfDomain -computerName $computerName -domain $domain){
 if (!(Test-JoinedToADomain)) {
 	Throw "$computerName not part of any domain"
 } else {
-	Remove-Computer <%= options[:parameters] %> -Workgroup 'WORKGROUP' -Verbose -Force -PassThru
+	#Remove-Computer <%= options[:parameters] %> -Workgroup 'WORKGROUP' -Verbose -Force -PassThru
```

Comment:
Can you please remove the commented code?

Reply:
Commented code removed.